compiler/qsc_frontend/src/resolve/tests.rs

```diff
@@ -4742,11 +4746,11 @@ fn export_namespace() {
                 operation item2() : Unit {}
             }
             namespace namespace8 {
-                export namespace7;
+                export item4;
```

Comment:
Wait a minute, does exporting a namespace actually do anything? I was under the impression you could only export callables and types.